TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -372,5 +372,29 @@ public LockTableStatement CreateLockTableStatement(CodeElementsParser.LockTableS
             var savepointName = context.Diagnostics == null ? new SymbolReference(new AlphanumericValue((Token)context.savepoint_name), SymbolType.SqlIdentifier) : null;
             return new ReleaseSavepointStatement(savepointName);
         }
+
+        public DropTableStatement CreateDropTableStatement(CodeElementsParser.DropTableStatementContext context)
+        {
+            if (context.tableOrAliasName() == null) return null;
```

Comment:
No. We should create CodeElement even when tableOrAliasName is missing. According to the unit test it's already working that way but this should be reflected in code here too.
I think CreateTableOrAliasName should allow null values for its param.